lms/djangoapps/grades/tasks.py

```diff
@@ -19,25 +21,51 @@
 from .signals.signals import SUBSECTION_SCORE_CHANGED
 from .transformer import GradesTransformer
 
+log = getLogger(__name__)
+
 
 @task(default_retry_delay=30, routing_key=settings.RECALCULATE_GRADES_ROUTING_KEY)
-def recalculate_subsection_grade(user_id, course_id, usage_id, only_if_higher):
+def recalculate_subsection_grade(user_id, course_id, usage_id, only_if_higher, grade, max_grade):
```

Comment:
Can we rename these variables to the nomenclature we have introduced in the grading layer?
So it would be `raw_earned` and `raw_possible`.